### PR TITLE
[OBMIN-349] TitleH3

### PR DESCRIPTION
**Pull request by Voloshkina**
Created subtitle component

### CODE REVIEW

**Review by Kyslytsia — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

File: src/components/subtitle/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as Styles from './styles';
+import { ISubtitle } from './types';
+
+export const Subtitle = ({ textTitle, hiddenStar, style }: ISubtitle) => {
+  return (
+    <Styles.TitleH3 style={style}>
+      {textTitle}&nbsp;
+      <Styles.Star hiddenStar={hiddenStar}>*</Styles.Star>
```
Comment (Wolshebnik) [inline src/components/subtitle/index.tsx:8]: {hiddenStar && <Styles........ />
here you can don't pass props

File: src/components/subtitle/styles.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  margin-bottom: 10px;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 26px;
+  font-weight: 600;
+  line-height: 40px;
+`;
+
+export const Star = styled.span<{ hiddenStar?: boolean }>`
+  ${({ hiddenStar, theme }) => css`
+    display: ${hiddenStar ? 'none' : 'inline-block'};
```
Comment (Wolshebnik) [inline src/components/subtitle/styles.ts:13]: then this line you can delete

File: src/components/subtitle/Subtitle.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+export const WithoutStar = Template.bind({});
+WithoutStar.args = {
+  hiddenStar: true,
+  textTitle: 'Опис товара',
+};
+
+export const OtherFont = Template.bind({});
+OtherFont.args = {
+  style: { fontFamily: 'Roboto' },
```
Comment (Wolshebnik) [inline src/components/subtitle/Subtitle.stories.tsx:29]: why is here this style
we use 2 fonts
Why do you give you the ability to add many fonts

**Review by Mish-hash — APPROVED** [review]

File: .storybook/preview-head.html [hunk 4]
```diff
@@ -3,4 +3,4 @@
 <link href="https://fonts.googleapis.com/css2?family=Caveat:wght@700&display=swap" rel="stylesheet" />
 <link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700&display=swap&subset=cyrillic"
   rel="stylesheet" />
-<link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&display=swap" rel="stylesheet" />
```
Comment (Wolshebnik) [inline .storybook/preview-head.html:6]: add empty line

**Review by Wolshebnik — COMMENTED** [review]
ok
how do you pass font family?

File: src/components/subtitle/types.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { CSSProperties } from 'react';
+
+export interface ISubtitle {
+  textTitle: string;
+  style?: CSSProperties;
+  hiddenStar?: true | false;
```
Comment (Wolshebnik) [inline src/components/subtitle/types.ts:6]: ))))
Maybe better hiddenStar?: boolean;

**Review by Wolshebnik — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Pull request merged**